SignalServiceKit/src/Messages/Interactions/OWSContact.h

```diff
@@ -160,21 +160,11 @@ NSString *NSStringForContactAddressType(OWSContactAddressType value);
 // TODO: Move to separate source file, rename to OWSContactConversion.
 @interface OWSContacts : NSObject
 
-#pragma mark - VCard Serialization
-
-+ (nullable CNContact *)systemContactForVCardData:(NSData *)data;
-+ (nullable NSData *)vCardDataForSystemContact:(CNContact *)systemContact;
-
 #pragma mark - System Contact Conversion
 
 + (nullable OWSContact *)contactForSystemContact:(CNContact *)systemContact;
```

Comment:
Please add a comment here indicating that these conversion methods deliberately skip avatars.

Reply:
Done!